nuxeo-services/nuxeo-platform-filemanager-api/src/main/java/org/nuxeo/ecm/platform/filemanager/api/FileImporterContext.java

```diff
@@ -92,6 +95,10 @@ public boolean isPersistDocument() {
         return persistDocument;
     }
 
+    public boolean isByPassSecurityCheck() {
```

Comment:
Please add `/** @since 11.1 */` as a Javadoc for this method.

Reply:
"since 11.1" even if the change is on the 10.10 branch ?

Reply:
Yes for backports our convention is to keep code identical too the `master` version.